Sources/AblyAssetTrackingInternal/AblyWrapper/DefaultAbly.swift

```diff
@@ -58,16 +74,11 @@ public class DefaultAbly: AblyCommon {
             
             let presenceEnterSuccess = { [weak self] in
                 guard let self = self else { return }
-                
-                self.logger.debug("Entered to channel [id: \(trackableId)] presence successfully", source: String(describing: Self.self))
```

Comment:
Why you've removed all those logger calls from this class? We should not be removing logs but simply sending them to our custom `LogHandler` :wink:

Reply:
It seemed to me like those logs would be more or less duplicates from what the ably-cocoa sends already. But maybe you're right, I'll re-add them. Especially since I see they are there in the Android code.

Reply:
Re-added in[1a7c51d](https://github.com/ably/ably-asset-tracking-swift/pull/362/commits/1a7c51d9d4e629d249d6e442babddec831ad837c)

I've also added a couple of error logs that were missing at the bottom of the file.